Source/SwiftLintFramework/Rules/DuplicateImportsRule.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  DuplicateImportsRule.swift
+//  SwiftLint
+//
+//  Created by Samuel Susla on 1/13/18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+extension Line {
```

Comment:
this extension should be `private` (and I would move it to the end of this file)

Reply:
Done